src/App.js

```diff
@@ -1,30 +1,77 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import './App.scss';
+import AirlineTable from './AirlineTable';
+import ObjectSelectorItem from './ObjectSelectorItem';
+import TimeSelector from './TimeSelector';
+
+class App extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.flightObjects =
+      [{
+        key: "departure",
+        name: "Departures"
+      }, {
+        key: "arrival",
+        name: "Arrivals"
+      }];
+
+    this.state = {
+      dayOffset: 0,
+      flightObject: this.flightObjects[0].key,
+      data: {}
+    }
+
+    this.updateData(0);
+  }
+
+  formatDate(date) {
+    let dd = date.getDate() + 1;
+    let mm = date.getMonth() + 1;
+    let yyyy = date.getFullYear();
+    if (dd < 10) {
```

Comment:
try to use - https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/String/padStart